src/main/kotlin/uk/gov/gdx/datashare/services/EventDataService.kt

```diff
@@ -123,6 +127,15 @@ class EventDataService(
     return mapEventNotification(event, acquirerSubscription, emptyList(), false)
   }
 
+  @Scheduled(fixedRate = 1, timeUnit = TimeUnit.HOURS)
```

Comment:
I'm not sure if this is the right place for this? but wasn't sure if it needed a new service

Reply:
I'd be tempted to have a separate dir for scheduled jobs. If not then some kind of EventStatisticsService

Reply:
What he said